src/main/zlatopolsky/chapter10/Ch1N10_52.java

```diff
@@ -0,0 +1,12 @@
+package zlatopolsky.chapter10;
+
+public class Ch1N10_52 {
+
+    public int reverseNumber(int n) {
+        if (n < 10) {
+            return n;
+        } else {
+            return Integer.parseInt(String.valueOf(n % 10) + reverseNumber(n / 10));
```

Comment:
//Why you use Integer.parseInt and String.valueOf? You have got an integer n,  you can use just n.

Reply:
this is necessary so that the numbers do not add up

Reply:
Ho-ho, you can multiply the number by 10 before adding the new digit

Reply:
Solving this task with numbers is preferable. Conversion to String and back to Integer is a cheat

Reply:
fix method reverse, use only mathematical operation